develop/webservices/WS_v1/WS_Methods_v1/GetAlarmHistory.md

```diff
@@ -6,9 +6,11 @@ uid: GetAlarmHistory
 
 Use this method to retrieve:
 
-- one specific alarm, or
+- one specific alarm,
 
-- all alarms in the alarm tree of a specific alarm.
+- all alarms in the alarm tree of a specific alarm, or
+
+- alarm details of a non-root alarm (Available from DataMiner 10.2.0 [CU13]/10.3.0 [CU1]/10.3.4).
```

Comment:
Same here, was also already part of the API call. Just was a description change.
The current description for the API call is:
_Get the alarm history for the specified alarm (optional full tree request). Use root alarm ID with requestFullTree for the details of a cleared non-root alarm._
Can be found here:
https://_your-dma_/api/v1/json.asmx?op=GetAlarmHistory